css/style.css

```diff
@@ -502,3 +515,330 @@ form {
 .form-group label {
   display: none;
 }
+
+hr {
+  display: none;
+}
+
+.first-last-name {
+  display: none;
+}
+
+.contact-illus-1,
+.contact-illus-2 {
+  display: none;
+}
```

Comment:
- [ ] Kindly remove the extra space after my recent works section,  fix the mobile version.
- [ ] Kindly align the entire section from the works section in the middle. As you have noticed that it's to the left side and the margin is not equal between the two sides.
- Your design: Kindly fix according to figma design.
<img width="365" alt="Screenshot 2022-03-24 at 11 06 43 PM" src="https://user-images.githubusercontent.com/70269507/160002678-9954b949-44af-4948-9e87-5fdf52835e65.png">
<img width="365" alt="Screenshot 2022-03-24 at 11 06 52 PM" src="https://user-images.githubusercontent.com/70269507/160002684-b0135b4f-515b-4a40-8009-81d52811c12d.png">
<img width="365" alt="Screenshot 2022-03-24 at 11 07 03 PM" src="https://user-images.githubusercontent.com/70269507/160002692-867db526-3ec3-4db4-ab84-c57fa026a945.png">
<img width="365" alt="Screenshot 2022-03-24 at 11 07 14 PM" src="https://user-images.githubusercontent.com/70269507/160002696-0450b4c4-af65-4d1b-a607-207329b33c40.png">